### DISCUSSION TITLE
Compatibility with Redshift

### DISCUSSION THREAD

**Discussion by ayazabbas** [cyrilgdn/terraform-provider-postgresql#111] [Q&A]
Is anyone using this provider to manage resources in Amazon Redshift? If so, has anyone faced any incompatibility issues? Just checking if this is a viable option for Redshift before I go down this route.

**Comment by cyrilgdn** [comment]
Hi @ayazabbas ,

I never used Redshift so I can't answer. But I know there were [opened issues](https://github.com/hashicorp/terraform-provider-postgresql/issues?q=is%3Aissue+redshift) for Redshift on this provider when it was still hosted by Hashicorp. You may want to take a look at these issues.
